src/haywire/http_server.c

```diff
@@ -111,6 +129,7 @@ void free_http_connection(http_connection* connection)
         free_http_request(connection->request);
     }
     
+    free(connection->response_buffers);
```

Comment:
@kellabyte unless I'm missing something, response buffers are allocated statically and not through malloc, so do they really need to be freed here?